src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerEditCase.java

```diff
@@ -78,14 +79,26 @@ private PageBuilder addEventConfig(ConfigBuilder<CaseData, State, UserRole> conf
     public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(CaseDetails<CaseData, State> details,
                                                                        CaseDetails<CaseData, State> beforeDetails) {
         CaseData data = details.getData();
-        State state = details.getState();
+        CaseData beforeData = beforeDetails.getData();
+        if (null != beforeData.getCicCase() && null != beforeData.getCicCase().getPartiesCIC()
+            && beforeData.getCicCase().getPartiesCIC().contains(PartiesCIC.REPRESENTATIVE)
```

Comment:
6 boolean conditions makes this a little hard to read.
Could you split out 3 conditions into a local helper method, that checks the 3 conditions (CicCase null, PartiesCIC null, PartiesCIC contains the enum), and pass in each CaseData to it?